packages/media-query/src/use-media-query.ts

```diff
@@ -4,38 +4,49 @@ import { useEffect, useLayoutEffect, useState } from "react"
 
 const useSafeLayoutEffect = isBrowser ? useLayoutEffect : useEffect
 
+export type UseMediaQueryOptions = {
+  fallback?: boolean | boolean[]
+  ssr?: boolean
+}
+
 /**
  * React hook that tracks state of a CSS media query
  *
  * @param query the media query to match
- * @param defaultValues the default values to match
+ * @param options the mediq query options { fallback, ssr }
```

Comment:
small typo on "media".  @segunadebayo Instead of passing `ssr` option on every hook, would it be worth considering passing `ssr` to `ChakraProvider`? Either way, thanks for your hard work!